db/migrate/20220514135214_add_post_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :post, null: false, foreign_key: true
+  end
+end
```

Comment:
Kindly add manually, another column to this migration file, to reference the user that created the comment.